packages/wonder-blocks-link/src/theme/default.ts

```diff
@@ -1,13 +1,14 @@
-import {border} from "@khanacademy/wonder-blocks-tokens";
+import {font} from "@khanacademy/wonder-blocks-tokens";
 
 export default {
     root: {
         font: {
             family: "inherit",
             weight: "inherit",
-        },
-        border: {
-            offset: border.offset.medium,
+            textDecoration: {
+                thickness: font.textDecoration.thickness,
+                offset: font.textDecoration.offset,
+            },
```

Comment:
Since the thickness and offset values are the same in default and TB, we can use the global tokens directly in `Link` without having component themed tokens! This will reduce the number of component css variables we have!

Reply:
Oh yes, good call!